### DISCUSSION TITLE
Possible to run on something like Google Coral?

### DISCUSSION THREAD

**Discussion by peteh** [openai/whisper#437] [General]
I'm relatively new to the whole topic and my initial searches via google did not really gave an answer. I'm running rhasspy on a pi4 and thought if it's possible to replace asr with whisper. However, the pi4 itself doesn't have the performance to run it anywhere in a useful timing. 

Is it possible to run whisper via google coral edge tpu/usb stick on something like a rasperry pi 4? Has anybody tried it?

**Comment by shervinemami** [comment]
I highly doubt anyone has tried it, but it does sound like a good idea to try. There is a discussion ticket where someone quantised the model to get upto a 3x speedup on CPUs. If you want to run on Coral, you'd need to do similar things but quantise using the exact methods & tools that Coral requires, since Coral can only run very limited types of operations, and can't process data that isn't quantised.

**Comment by TRusselo** [comment]
i also run a coral dual tpu on my unraid server.
would love support.
